backend/app/views/spree/admin/option_types/edit.html.erb

```diff
@@ -6,7 +6,9 @@
 <% content_for :page_actions do %>
   <li>
     <span id="new_add_option_value" data-hook>
-      <%= link_to_add_fields t('spree.add_option_value'), "tbody#option_values", class: 'btn btn-primary' %>
+      <%= link_to t('spree.add_option_value'), "javascript:",
```

Comment:
What about using a `button_tag`? This way we don't have to set that fake path with `javascript:`.

Reply:
ok. done.